params/version.go

```diff
@@ -21,20 +21,41 @@ import (
 )
 
 const (
+	// Version is the version of upstream geth
 	VersionMajor = 1          // Major version component of the current release
 	VersionMinor = 10         // Minor version component of the current release
 	VersionPatch = 22         // Patch version component of the current release
 	VersionMeta  = "unstable" // Version metadata to append to the version string
+
+	// OPVersion is the version of op-geth
+	OPVersionMajor = 0          // Major version component of the current release
+	OPVersionMinor = 1          // Minor version component of the current release
+	OPVersionPatch = 0          // Patch version component of the current release
+	OPVersionMeta  = "unstable" // Version metadata to append to the version string
 )
 
 // Version holds the textual version string.
 var Version = func() string {
-	return fmt.Sprintf("%d.%d.%d", VersionMajor, VersionMinor, VersionPatch)
+	return fmt.Sprintf("%d.%d.%d", OPVersionMajor, OPVersionMinor, OPVersionPatch)
 }()
 
 // VersionWithMeta holds the textual version string including the metadata.
 var VersionWithMeta = func() string {
 	v := Version
+	if OPVersionMeta != "" {
+		v += "-" + OPVersionMeta
```

Comment:
Would it make sense to use the build meta tag here, so that op version is not used for version precedence? So, `+` rather than `-`.

https://semver.org/#spec-item-10

Reply:
geth switches between `unstable` and `stable` using `-` here. It's possible to include the geth version with a `+` tag on our version without changing precedence.